jenkins/test-history/buckets.json

```diff
@@ -9,6 +9,7 @@
         "prefix": "rktnetes$"
     },
     "gs://kopeio-kubernetes-e2e/logs/": {
-        "prefix": "kopeio$"
+        "prefix": "kopeio$",
+        "sequential": false
```

Comment:
might be a good idea to add comments describing what each of these fields means?